yak_ros/src/yak_ros1_node.cpp

```diff
@@ -160,7 +200,7 @@ class OnlineFusionServer
 int main(int argc, char **argv)
 {
   ros::init(argc, argv, "tsdf_node");
-  ros::NodeHandle nh("tsdf_node");
+  ros::NodeHandle nh; // removed "tsdf_node" from node handle to keep service names cleaner
```

Comment:
@DavidMerzJr We should be getting the camera matrix from the camera_info_manager services of the camera driver, not from parameters set in the launch file.